_troubleshooting/destinations/aws-io-errors.md

```diff
@@ -0,0 +1,63 @@
+---
+title: AWS & Redshift I/O Connection Errors
+keywords: troubleshooting, destination, trouble, issue, help, error, errors, redshift, panoply, io, i/o
+permalink: /troubleshooting/destinations/aws-io-errors
+tags: [troubleshooting_destinations, troubleshooting_errors]
+
+summary: "While typically transient, I/O errors arise from connection issues between Stitch and your data warehouse. If persistent, these errors may be indicative of a larger issue."
+type: "redshift-destination, error"
+promote: "false"
+---
+{% include misc/data-files.html %}
+
+> An I/O error occurred while sending to the backend.
+
+{{ page.summary }} To ensure Stitch can continue loading your data, these errors should be addressed promptly.
+
+**Note**: While we've only seen this issue affect Amazon Redshift users, it is possible that PostgreSQL destinations may also be affected.
+
+---
+
+## I/O Errors, Explained {#io-error-explanation}
+
+I/O errors arise from connection issues between Stitch and your data warehouse. When Stitch connects to your destination to perform a connection check or load data, an I/O error can arise if the connection to the destination is severed. This is typically caused by a timeout issue.
+
+For example: Stitch attempts to load a large amount of data into your destination. Due to the data volume, Stitch's query takes a long time to run and, as a result, the connection becomes idle.
```

Comment:
change to "...as a result, the server closes the idle connection."

Reply:
Fixed.